api/src/pcapi/routes/pro/collective_offers.py

```diff
@@ -36,18 +37,33 @@
 @spectree_serialize(
     response_model=collective_offers_serialize.ListCollectiveOffersResponseModel,
     api=blueprint.pro_private_schema,
+    query_params_as_list=["status"],
 )
 def get_collective_offers(
     query: collective_offers_serialize.ListCollectiveOffersQueryModel,
 ) -> collective_offers_serialize.ListCollectiveOffersResponseModel:
+    if query.status:
+        assert isinstance(query.status, list)
+        # Only 1 status is permitted for the moment
+        # TODO: implement multi status passing
+        if len(query.status) != 1:
+            raise ApiErrors({"status": ["Only 1 status must be provided in query args"]}, status_code=400)
+        if len(query.status) == 1:
+            status_enum: CollectiveOfferDisplayedStatus = query.status[0]
+            status = status_enum.value
+        else:
+            status = None
+    else:
+        status = None
```

Comment:
Suite à mes commentaires précedents, est-ce qu'il ne serait pas plus simple d'écrire :
```suggestion
    status = None
    
    if query.status:
        assert isinstance(query.status, list)
        # Only 1 status is permitted for the moment
        # TODO: implement multi status passing
        if len(query.status) != 1:
            raise ApiErrors({"status": ["Only 1 status must be provided in query args"]}, status_code=400)
        
        status_enum: CollectiveOfferDisplayedStatus = query.status[0]
        status = status_enum.value
```